Tools/gdb/libpython.py

```diff
@@ -709,13 +709,18 @@ def write_repr(self, out, visited):
             pyop_value.write_repr(out, visited)
         out.write('}')
 
+    try:
+        _gdb_error = gdb.error
+    except AttributeError:
+        _gdb_error = RuntimeError
```

Comment:
@methane: " We can use just RuntimeError, maybe."

I prefer to use gdb.error when available to not catch a real RuntimeError bug by mistake.